src/main/java/org/radarcns/management/security/ClaimsTokenEnhancer.java

```diff
@@ -62,7 +63,7 @@ public OAuth2AccessToken enhance(OAuth2AccessToken accessToken,
                         .map(role -> role.getProject().getProjectName() + ":"
                                 + role.getAuthority().getName())
                         .collect(Collectors.toList());
-                additionalInfo.put(RadarAuthorization.ROLES_CLAIM, roles);
+                additionalInfo.put(JwtRadarToken.ROLES_CLAIM, roles);
```

Comment:
I'm wondering if this `additionalInfo` could be something that a `JwtRadarToken` generates. Perhaps with a builder?

Reply:
Interesting idea, i think it would make sense indeed to centralize the logic for creating this map in `JwtRadarToken`.